src/System.Reflection.Metadata/src/System/Reflection/Metadata/TypeSystem/TypeDefinition.cs

```diff
@@ -52,6 +52,19 @@ public TypeAttributes Attributes
             }
         }
 
+        public bool IsNested
```

Comment:
Seems like this could be simply: 
```
public bool IsNested => Attributes.IsNested();
```

Reply:
Agreed.

I tried to keep a consistent style and wasn't sure if expression bodied properties was appropriate. Even then the conditional I had is unnecessary.

I'll make the appropriate changes.